ionos_dev/marketplace/tmforum-api/values.yaml

```diff
@@ -0,0 +1,127 @@
+
+doc: 
+  defaultApi: party-catalog
+  defaultPath: /tmf-api/party/v4
+
+
+tm-forum-api:
+  ingress:
+    enabled: true
+    annotations:
+      cert-manager.io/cluster-issuer: letsencrypt-dev-issuer
+    # -- host to be used
+    hosts:
+      - host: tmforum.dome-marketplace-dev.org
+    tls:
+    - hosts:
+      - tmforum.dome-marketplace-dev.org
+      secretName: tmforum-tls 
+
+  apiProxy:
+    image:
+      tag: distroless-v1.27-latest
+    enabled: false
+    service: 
+      nameOverride: tmforum-api-proxy
+
+
+  ## configuration to be used by every api-deployment if nothing specific is provided.
+  defaultConfig:
+  
+    # -- configuration to be used for the image of the container
+    image:
+      # -- repository to get the container from
+      repository: quay.io/fiware
+      # -- tag to be used, most of the time the apis will use the same version
+      tag: 0.15.0
+      # -- pull policy to be used
+      pullPolicy: IfNotPresent
+
+    # --  ngsi-ld broker connection information
+    ngsiLd:
+      # -- timeout for requests ot the broker
+      readTimeout: 30s
+      # -- base path for the ngsi-ld api
+      path: ngsi-ld/v1
+      # -- address of the broker
+      url: http://scorpio:1026
+
+    logLevel: INFO
+    contextUrl: https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld
+
+    additionalEnvVars:
+      - name: GENERAL_NGSILDORQUERYVALUE
+        value: ","
+      - name: GENERAL_NGSILDORQUERYKEY
+        value: ","
+      - name: GENERAL_ENCLOSEQUERY
+        value: "false"
+
+  # -- list of apis to be deployed
+  # -- every api needs to define a name and image. Basepath is required when routes or ingress will be enabled. Beside that, they can overwrite the default-config by using the same keys.
+  # -- be aware: when you change the image repositrory or the tag for an api, you have to provide both values for the changes to take effect
+  apis:
+    - name: party-catalog
+      image: tmforum-party-catalog
+      basePath: /
+      specUrl: https://tmf-open-api-table-documents.s3.eu-west-1.amazonaws.com/OpenApiTable/4.0.0/swagger/TMF632-Party-v4.0.0.swagger.json
```

Comment:
Is the specUrl still used? the swagger file is now part of the TMForum API project